dpe/src/commands/derive_context.rs

```diff
@@ -196,6 +211,8 @@ impl CommandExecution for DeriveContextCmd {
             || (!dpe.support.internal_dice() && self.uses_internal_dice_input())
             || (!dpe.support.retain_parent_context() && self.retains_parent())
             || (!dpe.support.x509() && self.allows_x509())
+            || (!dpe.support.cdi_export()
+                && (self.allows_create_certificate() && self.allows_exports_cdi()))
```

Comment:
Might also need to check that if `dpe.support.cdi_export()` is true then `dpe.support.x509()` is also true.

Reply:
I think technically we could create a CSR with IsCA = true

Reply:
Hmm, in that case I'll omit this check, if this invariant does occur it will fail (eventually)